rct229/rules/section6.py

```diff
@@ -136,9 +136,9 @@ def get_calc_vals(self, context, data=None):
                             )
 
                 if building_segment_uses_building_area_method:
-                    building_segment_allowable_lpd = table_G3_8_lpd(
+                    building_segment_allowable_lpd = table_G3_8_lookup(
                         building_area_type=building_segment_lighting_building_area_type
-                    )
+                    ).lpd
```

Comment:
Since the return value for table_G3_8_lookup() is a dict, line 141 needs to be:
`)["lpd"]`